project/Build.scala

```diff
@@ -28,10 +33,26 @@ object ScalaMeterBuild extends MechaRepoBuild {
       publish <<= streams.map(_.log.info("Publishing to Sonatype is disabled since the \"" + publishUser + "\" and/or \"" + publishPass + "\" environment variables are not set."))
   })
 
+  val releasePluginSettings =  releaseSettings ++ Seq(
+    releaseBranchName := s"version/${(version in ThisBuild).value}",
+    commitMessage <<= (version in ThisBuild) map (v => "Set version to %s" format v),
```

Comment:
Since SBT runs on Scala 2.10, you should be able to use string interpolation here with no problems - this is preferred to using `format`.